docs/guides/core_client.md

```diff
@@ -313,6 +512,23 @@ Optional command line options for the public/user decryption command are:
 
  __NOTE__: If the ciphertext is provided by file, then only the optional arguments `-b`/`--batch-size <BATCH_SIZE>` and `-n`/`--num-requests <NUM_REQUESTS>` are supported.
 
+### Custodian context
+
+In order to be able to do custodian based backup and recovery, the KMS nodes need to know the public keys of the custodians which will be able to help it recover. This is handled through custodian contexts. 
+For custodian based backup we currently only support a single active custodian context. This there will only exist one set of custodians under which a backup can be constructed. Whenever a new custodian context is made, this will replace the old context as the current backup method. 
```

Comment:
```suggestion
For custodian based backup we currently only support a single active custodian context. This means there will only exist one set of custodians under which a backup can be constructed. Whenever a new custodian context is made, this will replace the old context as the current backup method. 
```